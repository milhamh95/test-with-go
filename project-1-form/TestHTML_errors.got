
	<label>Email Address</label>
	<input
		class=""
		type="email"
		name="EmailAddress"
		placeholder="you@domain.com"
		value="email@taken.com">
	
	<label>Password</label>
	<input
		class=""
		type="password"
		name="Password"
		placeholder="Password"
		value="badpw">
	