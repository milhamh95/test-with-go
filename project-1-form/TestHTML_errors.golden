
	<label>Email Address</label>
	<input
		class="border-red"
		type="email"
		name="EmailAddress"
		placeholder="you@domain.com"
		value="email@taken.com">
	<p class="text-red text-xs italic">Email address is already taken</p>

	<label>Password</label>
	<input
		class="border-red"
		type="password"
		name="Password"
		placeholder="Password"
		value="badpw">
	<p class="text-red text-xs italic">Password must be between 201 and 210 characters</p>
	<p class="text-red text-xs italic">Password must contain a greek letter</p>
	<p class="text-red text-xs italic">Password must be a palindrome</p>
	<p class="text-red text-xs italic">Password must contain an emoji</p>
	